apps/ui/src/components/ProposalResults.vue

```diff
@@ -45,83 +45,127 @@ const results = computed(() =>
 </script>
 
 <template>
-  <div v-if="proposal.type !== 'basic'">
+  <div
+    v-if="!!props.proposal.privacy && !props.proposal.completed"
+    class="text-center p-3.5 leading-5"
```

Comment:
There is already padding on parent, no need for additional padding on x-axis here.